README.md

```diff
@@ -27,8 +27,10 @@ nginx needs to be reloaded by entering the container and running
 ## Build and run locally
 Run `./scripts/dev.sh`. This runs up the proxy along with the apis and portals, in order to manually test links, logins etc. 
 The test user with email `test.user@example.com` and password `password` is added by default.
-Optionally include 'data' parameter (`./scripts/dev.sh data`) to include generating Montagu test data. Orderly test data 
-is always generated.
+Optionally include 'data' parameter (`./scripts/dev.sh data`) to include generating Montagu test data. 
+
+Demo orderly data is expected to be found in `../packit/demos/orderly` so it is necessary to have 
+[packit](https://github.com/mrc-ide/packit) cloned in the same parent directory as this repo.
```

Comment:
This isn't ideal but I think the alternative would be to have packit as a submodule, which I also wouldn't feel good about.